.github/workflows/release-workflow.yml

```diff
@@ -64,13 +64,14 @@ jobs:
         run: |
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 29 to 29]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->You'll need the SHA here too<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/337?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->